utils/getClaimStatus.tsx

```diff
@@ -7,66 +7,115 @@ import { ClaimStatusContent, I18nString, TextOptionalLink, TransLineProps } from
 import { ScenarioType } from './getScenarioContent'
 import getUrl, { UrlType } from './getUrl'
 
+type StepType = 'your-next-steps' | 'edd-next-steps'
+
+interface ClaimStatusScenarioJson {
+  heading: string
+  summary: TextOptionalLink
+  'your-next-steps': TextOptionalLink[]
+  'edd-next-steps': TextOptionalLink[]
+}
```

Comment:
I introduced some stronger types here so I was able to both avoid the very loosely typed [recursion approach](https://github.com/cagov/ui-claim-tracker/blob/18610707d4d7a6e6367b0e45f079f6a06cfba2ca/utils/getClaimStatus.tsx) (check out `getChildJson()`) I was trying out and end up with slightly tighter typing than in PR #314.